inc/toolbox.class.php

```diff
@@ -723,26 +723,16 @@ static function setDebugMode($mode = null, $debug_sql = null, $debug_vars = null
 
       // If debug mode activated : display some information
       if ($_SESSION['glpi_use_mode'] == Session::DEBUG_MODE) {
-         // display_errors only need for for E_ERROR, E_PARSE, ... which cannot be catched
          // Recommended development settings
          ini_set('display_errors', 'On');
-         error_reporting(E_ALL | E_STRICT);
+         error_reporting(E_ALL);
```

Comment:
E_STRICT is part of E_ALL since PHP 5.4 (see https://www.php.net/manual/fr/function.error-reporting.php#refsect1-function.error-reporting-changelog)